docs/ops/opset8.md

```diff
@@ -44,6 +44,7 @@ declared in `namespace opset8`.
 * [DeformablePSROIPooling](detection/DeformablePSROIPooling_1.md)
 * [DepthToSpace](movement/DepthToSpace_1.md)
 * [DetectionOutput](detection/DetectionOutput_1.md)
+* [DetectionOutput](detection/DetectionOutput_8.md)
```

Comment:
Do we really need to add both versions of DetectionOutput operation to opset8.md file? If yes, we should specify versions in names to avoid any confuses.

Reply:
Done